source/change-streams/tests/unified/change-streams.yml

```diff
@@ -227,16 +227,21 @@ tests:
   - description: "to field is set in a rename change event"
     runOnRequirements:
       - minServerVersion: "4.0.1"
+        topologies: [ replicaset ]
     operations:
       - name: createChangeStream
         object: *collection0
         arguments:
           pipeline: []
         saveResultAsEntity: &changeStream0 changeStream0
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: &collection1 collection1
```

Comment:
I'm not sure what tests introduced this, but I think it's preferable to use separate references for entity IDs and the database and collection names.

Unified tests have typically done something like the following: https://github.com/mongodb/specifications/blob/98db467cb8f2f7bf50e79fc563e59701a8b3e3bf/source/unified-test-format/tests/valid-pass/poc-change-streams.yml#L14-L21

I didn't bother to mention this in https://github.com/mongodb/specifications/pull/1156 but it applies there as well.

Reply:
Ah, makes sense.  And yes, I'd prefer to do that as a single large batch in that PR.